### PR TITLE
Client metrics are defined using metric-schema

### PR DESCRIPTION
**Pull request by carterkozak**
https://github.com/palantir/metric-schema

==COMMIT_MSG==
Client metrics are defined using metric-schema
==COMMIT_MSG==

### CODE REVIEW

File: okhttp-clients/src/main/metrics/okhttp-metrics.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      request-permit.slow-acquire:
+        type: timer
+        docs: Concurrency limiter request permit slow acquisition time.
+      request-permit.leak-suspected:
+        type: meter
+        docs: Rate of suspected request permit leaks.
+      request-permit.slow-acquire-tagged:
+        type: timer
+        tags: [serviceClass]
+        docs: Concurrency limiter request permit slow acquisition time. Idential to `slow-acquire`, but tagged by service.
```
Comment (carterkozak) [inline okhttp-clients/src/main/metrics/okhttp-metrics.yml]: I'd appreciate if someone with more context could improve the documentation for these, I wasn't able to find any existing documentation.

Comment (iamdanfox) [reply okhttp-clients/src/main/metrics/okhttp-metrics.yml]: Have pushed some extra sentences :)

**Review by iamdanfox — APPROVED** [review]
LGTM, might need to also put your own `:+1:` on this

**Comment by carterkozak** [comment]
Fantastic, thanks @iamdanfox! :+1:

**Pull request merged**